serif_shared/src/commonMain/kotlin/xyz/room409/serif/serif_shared/MatrixClient.kt

```diff
@@ -67,7 +67,26 @@ class MatrixSession(val client: HttpClient, val server: String, val user: String
 
     fun getRoomEvents(id: String) = synchronized(this) { sync_response!!.rooms.join[id]!!.timeline.events }
 
+    fun createRoom(name: String, room_alias_name: String, topic: String): Outcome<String> {
+        try {
+            val result = runBlocking {
+                val creation_confirmation =
+                    client.post<EventIdResponse>("$server/_matrix/client/r0/createRoom?access_token=$access_token") {
```

Comment:
Probably should either send the transaction id, if it is accepted, or get rid of the transaction id increment+db update down below.